go-controller/pkg/ovn/egressfirewall_dns.go

```diff
@@ -178,6 +188,10 @@ func (e *EgressDNS) Shutdown() {
 	close(e.stopChan)
 }
 
-func (e *EgressDNS) signalAdded(dnsName string) {
-	e.added <- dnsName
+func (e *EgressDNS) signalAdded() {
```

Comment:
@JacobTanenbaum @trozet @squeed is it a problem to block sending, now that the only caller is run from a goroutine? Just folding the `e.added <- struct{}{}` into addToDNS() would be simpler and clearer, and just let it block, so that all the goroutines pile up on the stack and are visible in a pprof dump if needed.

Reply:
@dcbw there is really not any reason to block, it would possibly add unneeded calls to `GetNextQueryTime()` we could fold the signaling into `addToDNS()` but I would still do it in a way that was not blocking

Reply:
I'm good either way. It might make sense to add some debug logging though like we do in the openflow code:
```
func (c *openflowManager) requestFlowSync() {
	select {
	case c.flowChan <- struct{}{}:
		klog.V(5).Infof("Gateway OpenFlow sync requested")
	default:
		klog.V(5).Infof("Gateway OpenFlow sync already requested")
	}
}
```

Reply:
@trozet @dcbw added the debug logging that was suggested

Reply:
@trozet @JacobTanenbaum I think it's just simpler and clearer to pick one of (1) `go e.addToDNS(dnsName)` or (2) non-blocking additions to the `added` channel. We don't need both do we?